hack/build/repos.bzl

```diff
@@ -4727,8 +4727,8 @@ def go_repositories():
         build_file_generation = "on",
         build_file_proto_mode = "disable",
         importpath = "go.uber.org/zap",
-        sum = "h1:CSUJ2mjFszzEWt4CdKISEuChVIXGBn3lAPwkRGyVrc4=",
-        version = "v1.18.1",
+        sum = "h1:mZQZefskPPCMIBCSEH0v2/iUqqLrYtaeqwD6FUGUnFE=",
+        version = "v1.19.0",
```

Comment:
All automated changes I assume.

Reply:
Yes, from updating kube libraries in `go.mod`. I didn't do anything by hand in this file